benchmarks/yamauchi_takei_2016_anelasticity/plot_output/gmt.conf

```diff
@@ -0,0 +1,11 @@
+#
+# GMT 5.4.4 Defaults file
```

Comment:
Will those who run your benchmark need to use these GMT files?

Reply:
Nope, have removed